pytorch/buildspec-sagemaker.yml

```diff
@@ -0,0 +1,104 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
Are we splitting the buildspec for old framework versions as well?

Reply:
We can split older versions as needed, but as of now, it only applies for current and future versions.